rules/windows/builtin/security/win_security_windows_defender_exclusions_write_deleted.yml

```diff
@@ -1,12 +1,18 @@
-title: Windows Defender Exclusion Set
-id: e9c8808f-4cfb-4ba9-97d4-e5f3beaa244d
+title: Windows Defender Exclusion Deleted
+id: a33f8808-2812-4373-ae95-8cfb82134978
+related:
+    - id: e9c8808f-4cfb-4ba9-97d4-e5f3beaa244d
+      type: derived
+    - id: 46a68649-f218-4f86-aea1-16a759d81820
+      type: derived
 status: test
-description: Detects scenarios where an Windows Defender exclusion was added in registry where an entity would want to bypass antivirus scanning from Windows Defender
+description: |
+    Detects when a Windows Defender exclusion has been deleted. This could indicate an attacker trying to delete its tracks by removing the added exclusions
```

Comment:
```suggestion
    Detects when a Windows Defender exclusion has been deleted. This could indicate an attacker trying to delete their tracks by removing the added exclusions
```